Projects/Features/MyInfoFeature/Sources/ViewControllers/MyInfoViewController.swift

```diff
@@ -0,0 +1,11 @@
+import BaseFeature
+import UIKit
+
+/// TODO:
+public class MyInfoViewController: UIViewController {
+    override public func viewDidLoad() {
```

Comment:
```suggestion
final class MyInfoViewController: UIViewController {
    override func viewDidLoad() {
```
pubilc으로 두지 않아도 될거같아요!

Reply:
아 이게 MainTab에서 접근하려면 public이어야 하더라구요

Reply:
타입 캐스팅할 때 접근하더라구요

Reply:
타입 캐스팅이 왜 이뤄지나요?

Reply:
이제 public지워도 되겠네요